tooling/ast_fuzzer/src/program/func.rs

```diff
@@ -637,6 +660,16 @@ impl<'a> FunctionContext<'a> {
         Ok(Some(Expression::Assign(Assign { lvalue, expression: Box::new(expr) })))
     }
 
+    /// Break out of a loop.
+    fn gen_break(&mut self) -> Expression {
+        Expression::Break
+    }
```

Comment:
I reckon we can just return `Expression::Break` in `gen_stmt` directly, and then these to `gen_` methods don't have to be the exceptions that don't take an `Unstructured`.